proposals/0008-multiwriter.md

```diff
@@ -300,6 +300,11 @@ section for more details.
 # Examples
 [examples]: #examples
 
+The examples below assume that a DEP-0007 `HypercoreHeader` message (with
+protocol `hyperdb`) has been written for all feeds. Feed entry indexes (aka,
+sequence numbers) are zero-indexed, but with the header message at index 0,
+hyperdb entries start at 1.
```

Comment:
This sentence is rather long, and contains a lot of important details. Perhaps splitting it into two sentences might help.

Reply:
It is two sentences! But I rewrote it.